data-science-pipelines-operator/samples/custom-configs/dspa.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: datasciencepipelinesapplications.opendatahub.io/v1alpha1
+kind: DataSciencePipelinesApplication
+metadata:
+  name: sample
+spec:
+  apiServer:
+    deploy: true
+    image: quay.io/modh/odh-ml-pipelines-api-server-container:v1.18.0-8
```

Comment:
Are the samples deployed with RHODS? If yes, the images cannot use images from quay.io (except when having a permit by prodsec/legal). You'll find more info at RHODS-6299

Reply:
everything in the `samples` folder are not deployed in rhods, they are mostly documentation / testing resources, hence all s3 keys, db passwords, images, etc mentioned within this folder are not production related

having said that, if this raises concerns I can proceed to removing them

Reply:
I removed samples folder as it's not required for rhods deployment.